src/main/java/io/xpire/commons/core/Messages.java

```diff
@@ -13,5 +13,7 @@
     public static final String MESSAGE_INVALID_TAGS = "Tags are not in the item specified!";
     public static final String MESSAGE_UNKNOWN_DELETE_MODE = "Unknown Delete mode.";
     public static final String MESSAGE_SUGGESTIONS = " Did you mean %s?";
-
+    public static final String MESSAGE_REMINDER_THRESHOLD_EXCEEDED =
```

Comment:
It may not be clear to the user what it means for a reminder threshold to be exceeded. Perhaps you could simply state "Reminder for item's expiry date starts today!"

Reply:
Alright!